website/website/add_youtuber.py

```diff
@@ -0,0 +1,133 @@
+import os
+from urllib.parse import unquote, urlparse
+
+from googleapiclient.discovery import build
+
+
+class YoutubeApi:
+    """A class to interact with the YouTube API."""
+
+    #: (str): The API key used to authenticate requests to the YouTube API.
+    API_KEY = None
+
+    #: (str): The URL of the YouTube channel.
+    channel_url = None
+
+    #: (str): The custom name of the channel.
+    username = None
+
+    #: (str): The identifier of the channel.
+    channel_id = None
+
+    #: (str): The title of the channel.
+    channel_title = None
+
+    #: (str): The full description of the channel.
+    channel_description = None
+
+    def __init__(self, API_KEY, channel_url) -> None:
+        self.API_KEY = os.environ.get('API_KEY')
+        self.channel_url = channel_url
+        self.username = None
```

Comment:
No need in setting the values as None in constructor, if the value is equal to the one on the class level:

        self.username = None
        self.channel_id = None
        self.channel_title = None
        self.channel_description = None